packages/ds-medicare-gov/docs/src/pages/patterns/consistent-header.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Consistent header
+usage: |
+  The Consistent Header is a shared component available for all of medicage.gov. The header has two main states and three screen size ranges. The information below provides guidance for the various versions of the header so you can easily integrate it into any experience.
+---
+
+For detailed information on the header contents, see [this Confluence page](https://confluence.cms.gov/display/MAX/%5BMAX%5D+Consistent+Header+Contents).
```

Comment:
Two things here:
1. It seems a little odd that we are linking to a Confluence page that requires a login on our doc site. Is that a practice we do other places?
2. If we do want to link to confluence here, can we get an updated link? When I follow this URL, I get a `Page not found`

Reply:
1. I think it's odd as well. This is the first attempt to get some of the consistent header into the Doc site and is very MVP
2. Link updated!